### PR TITLE
chore(docs): update vocs to alpha.61 to add code copy buttons

### LINKED ISSUE #1256: Feature Request: Add Copy Button to Code Snippets in OnchainKit Documentation
### **Issue Summary**
Currently, the OnchainKit documentation lacks a copy button for code snippets, making it inconvenient for developers to quickly copy and paste the provided examples into their projects. This feature request proposes adding a copy button to all code blocks to enhance usability and improve the overall developer experience.

### **Motivation**
- **Increased Efficiency**: A copy button would allow developers to quickly copy the code without having to manually select text, reducing time and errors.
- **Improved Developer Experience**: Many popular documentation sites include this feature, and adding it to OnchainKit will bring the docs in line with modern standards.

### **Proposed Solution**
- Implement a copy-to-clipboard button for all code snippets in the documentation.
- The button should appear at the top-right corner of the code block.
- When clicked, it should copy the entire code snippet to the user’s clipboard and provide a success notification (e.g., "Copied!").

### **Acceptance Criteria**
- [ ] A copy button is visible on all code snippets in the OnchainKit documentation.
- [ ] Clicking the button copies the entire code block to the user’s clipboard.
- [ ] A small notification (e.g., "Copied!") confirms the action.

### Describe alternatives you've considered.

_No response_

### PR DESCRIPTION
**Pull request by Elishaokon13**
## Summary

This PR updates the `vocs` package from `alpha.58` to `alpha.61` to introduce the "Copy to Clipboard" button for code snippets in the OnchainKit documentation.

## Issue Reference

Closes #1256.

## Changes

- **Update**: Bumps `vocs` version from `alpha.58` to `alpha.61`.
- **Feature**: Adds a copy button to all code snippets within the OnchainKit documentation to improve the developer experience.

## Why?

The new version of `vocs` brings better usability for developers by enabling quick copying of code snippets. This change will streamline the process of working with the documentation and reduce friction in development workflows.

## Testing

- Verified that the documentation renders correctly with the updated version.
- Checked that the copy button appears on all code snippets and works as expected.

## Screenshots

Screenshot 2024-10-05 at 05 42 59

## Checklist

- [ ] Update documentation as needed.
- [x] Ensure no breaking changes with the version update.
- [x] Tested functionality locally.

### CODE REVIEW

**Comment by Elishaokon13** [comment]
@yuga-cb please review

File: site/docs/pages/transaction/transaction.mdx [hunk 1]
```diff
@@ -310,6 +310,7 @@ Obtain a Paymaster and Bundler endpoint from the [Coinbase Developer Platform](h
 import { Transaction, TransactionButton, TransactionSponsor } from "@coinbase/onchainkit/transaction"
 // ---cut-before---
 // ommited for brevity
+// @errors: 2322
```
Comment (alessey) [inline site/docs/pages/transaction/transaction.mdx:313]: think you'll need @noErrors: 2322 to completely suppress the error

Comment (Elishaokon13) [reply site/docs/pages/transaction/transaction.mdx]: Just did

**Comment by Elishaokon13** [comment]
Hi @alessey 
I've fixed the issue

**Pull request closed without merging**